### PR TITLE
[docs]: add docs for e2e

### PR DESCRIPTION
**Pull request by joshuaellis**
### What does it do?

* Adds some docs about the e2e setup as is today

### Why is it needed?

* Docs are good for you

### CODE REVIEW

**Review by Convly — COMMENTED** [review]
lgtm, I would just add a warning about the fact DTS doesn't export nor import admin users, API tokens, admin roles + relations between entities and admin users (createdBy, updatedBy)

**Comment by joshuaellis** [comment]
> relations between entities

So just so im clear (cause we'll need to overcome this at some point) if you have a `Menu` with `3 Plates` and export / import the data `Menu` will have no plates in the new instance?

**Review by Convly — APPROVED** [review]
lgtm

Also, admin entities (users, roles, api tokens) are not exported only when using the CLI. If you're making a custom export script by plugin providers together, everything will be exported.

File: docs/docs/testing/e2e/app-template.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+#### Usage
+
+```ts
+import { test } from '@playwright/test';
+
+test.beforeEach(async ({ page }) => {
+  await page.request.fetch('http://localhost:1337/api/database-dump', {
+    method: 'POST',
+  });
+});
```
Comment (gu-stav) [inline docs/docs/testing/e2e/app-template.md:35]: Isn't that outdated now that we are using DEITS directly? In any case the URL should be `/api/database` :)

Comment (joshuaellis) [reply docs/docs/testing/e2e/app-template.md:35]: I'd like to keep it just incase, it's very useful! i've amended the docs, looking at the route it should be `database/dump` 👍🏼

**Review by gu-stav — APPROVED** [review]
Thank you 🌻

**Pull request merged**